game/scripts/inventory/item_data.gd

```diff
@@ -1,7 +1,16 @@
 extends Resource
 class_name ItemData
 
+enum InteractionType {
+	INTERACT,
+	PICK_UP,
+	EQUIP,
+	NOT_INTERACTABLE,
+}
```

Comment:
Nice! I wonder if this could be saved / abstracted somewhere else such that a non-item can have these interaction types. Ie, a locked door can have some interaction. 

fine for now tho.

Reply:
Agreed